src/content/community/research/index.md

```diff
@@ -0,0 +1,442 @@
+
+#### Background reading
+
+[Tooling by programming language](/developers/docs/programming-languages/)
+
+[Developer Frameworks](/developers/docs/frameworks/)
+
+[Consensus developer tools list](https://github.com/ConsenSys/ethereum-developer-tools-list)
+
+[Token standards](/developers/docs/standards/tokens/)
+
+[Biastek: Ethereum tools](https://biastek.com/ethereum-tools/)
+
+[CryptoDevHub: EVM Tools](https://cryptodevhub.io/wiki/ethereum-virtual-machine-tools)
+
+#### Recent research
+
+[Eth R&D Discord Consensus Tooling channel](https://discordapp.com/channels/595666850260713488/746343380900118528)
+
+### Oracles
+
+Oracles are used to import off-chain data on-chain in a permissionless and decentralized way. This enables dapps to be reactive to real world phenomena such as price fluctuations in real-world assets, events in off-chain apps or even changes in the weather.
+
+#### Background reading
+
+[Introduction to Oracles](/developers/docs/oracles/)
+
+#### Recent Research
+
+[Survey of blockchain oracles](https://arxiv.org/pdf/2004.07140.pdf)
+
+[Chainlink white paper](https://chain.link/whitepaper)
+
+### App security
+
+Hacks on Ethereum nearly always exploit vulnerabilities in individual applications rather than the base protocol. Hackers and app developers are locked in an arms race to develop new attacks and defenses. This means there is always important research and development required to keep apps safe from hacks.
+
+#### Background reading
+
+[Wormhole exploit report](https://blog.chainalysis.com/reports/wormhole-hack-february-2022/)
+
+[List of Ethereum contract hack post-mortems](https://forum.openzeppelin.com/t/list-of-ethereum-smart-contracts-post-mortems/1191)
+
+[Rekt News](https://twitter.com/RektHQ?s=20&t=3otjYQdM9Bqk8k3n1a1Adg)
+
+#### Recent research
+
+[ethresear.ch Applications](https://ethresear.ch/c/applications/18)
+
+### Technology stack
+
+Apps on Ethereum are rarely fully decentralized because they rely on some centralized tooling or infrastructure. Decentralizing the full tech stack is an important research area.
+
+#### Background reading
+
+[Coinbase: Intro to Web3 Stack](https://blog.coinbase.com/a-simple-guide-to-the-web3-stack-785240e557f0)
+
+[Ethereum stack](/developers/docs/ethereum-stack/)
+
+[Introduction to smart contracts](/developers/docs/smart-contracts/)
+
+[Introduction to decentralized storage](/developers/docs/storage/)
+
+#### Recent research
+
+[Smart contract composability](/developers/docs/smart-contracts/composability/)
+
+## Sources of Funding
+
+You can get involved with Ethereum research and get paid for it! For example, the [Ethereum Foundation](/foundation/) recently ran an [Academic Grants](/community/grants/academic-grants-round/) funding round. Information on active and upcoming funding opportunities can be found on the [Ethereum grants page](/community/grants/).
```

Comment:
Using the entire relevant context of text for links is generally better for UX/a11y

```suggestion
You can get involved with Ethereum research and get paid for it! For example, [the Ethereum Foundation](/foundation/) recently ran an [Academic Grants funding round](/community/grants/academic-grants-round/). You can find information on active and upcoming funding opportunities on [the Ethereum grants page](/community/grants/).
```